src/SprykerEco/Yves/Payolution/Theme/default/installment.twig

```diff
@@ -1,41 +0,0 @@
-<div class="payolution-form payolution-installment-form payment-form small-12 medium-9 large-6" id="payolutionInstallment">
```

Comment:
Do not delete these old files. You can leave them as BC. @tolerants what do you think?

Reply:
Agree, but suggest to add a comment, that these files are deprecated.